helpers.go

```diff
@@ -0,0 +1,12 @@
+package metrics
+
+import "github.com/sirupsen/logrus"
+
+func AnyTagsToLogrusFields(tags AnyTags) logrus.Fields {
```

Comment:
I think this's too long name for a helper. Imagine how it will look in the code.

Reply:
yes, I agree with you, I'll fix it. and I found another issue with this helper, don't merge it!